new_client/src/components/auth/Signin.jsx

```diff
@@ -58,15 +59,17 @@ function Signin(props) {
 	useEffect(() => {
 		if (error.id === LOGIN_FAIL) {
 			setMsg(error.msg.msg);
+			alert(msg);
 		}
 		else {
 			setMsg(null);
 		}
 
 		if (isAuthenticated) {
 			// Work here if auth is successful
+			history.push("/");
 		}
-	}, [error, isAuthenticated]);
+	}, [error, isAuthenticated, history, msg]);
```

Comment:
Are you sure we need to to add `history` and `msg` here?

Reply:
The msg was unneeded and removed it, history is required ig